net/mosquitto/Portfile

```diff
@@ -4,8 +4,8 @@ PortSystem          1.0
 PortGroup           cmake 1.1
 
 name                mosquitto
-version             1.5.8
-revision            2
+version             2.0.9
+revision            1
```

Comment:
then updating the `version` the `revision` field should be reset to `0`

Reply:
Done!